views/apartments/index.erb

```diff
@@ -1,5 +1,16 @@
-<ul>
-  <li><a href='/apartments/1'>Apartment 1</a></li>
-  <li><a href='/apartments/2'>Apartment 2</a></li>
-  <li><a href='/apartments/3'>Apartment 3</a></li>
-</ul>
+<h1>
+  Apartments
+  <a href="/apartments/new">(+)</a>
+</h1>
+<section class="apartments">
+  <ul>
+    <% @apartments.each do |apartment| %>
+    <li>
+      <a href='/apartments/<%= apartment.id %>'>
+        <img src="<%= apartment.image_url(32,32) %>" />
+        <%= apartment %>
```

Comment:
If you just print out `apartment`, you'll probably get something like `<Apartment::ActiveRecord::Base blahblahblah>`. I think you meant to specify `apartment.address`?